### PR TITLE
Stuf ZDS integration creates hardcoded document

### LINKED ISSUE #1769: StUF-ZDS integration creates documents with hardcoded vertrouwelijkheidsaanduiding 'VERTROUWELIJK'
stuf/stuf_zds/templates/stuf_zds/soap/voegZaakdocumentToe.xml contains a hardcoded value 'VERTROUWELIJK', which indicates that documents created via StUF-ZDS are always confidential.

As a default this is fine to ensure backwardscompatibility, however this value should be configurable via the StUF-ZDS Registration plugin (StufZDSRegistration -> ZaakOptionsSerializer) to allow the staff user to specify a different level of confidentialy. Make use of VertrouwelijkheidsAanduidingen.choices like in the ZGW API Registration plugin.

### PR DESCRIPTION
**Pull request by dariomory**
Fixes #1769

### CODE REVIEW

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
Your test assertions have lowercased values, while the situation before this patch had uppercase values. How did you confirm that lowercase values are correct?

File: src/openforms/registrations/contrib/stuf_zds/plugin.py [hunk 1]
```diff
@@ -55,6 +56,14 @@ class ZaakOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serializer):
         help_text=_("Documenttype description for newly created zaken in StUF-ZDS"),
     )
 
+    zds_vertrouwelijkheidsaanduiding = serializers.ChoiceField(
+        choices=VertrouwelijkheidsAanduidingen.choices,
+        default=VertrouwelijkheidsAanduidingen.vertrouwelijk,
```
[the remark this answers is not in the document]
Comment (dariomory) [reply src/openforms/registrations/contrib/stuf_zds/plugin.py:61]: Suggested by @alextreme. See https://github.com/open-formulieren/open-forms/issues/1769#issue-1301718751

**Review by sergei-maertens — APPROVED** [review]
Tweaked the tests to ensure we can properly upgrade without breaking existing forms (and made them pass too).

**Comment by sergei-maertens** [comment]
Backports `stable/1.0.x`: c981cbe83245c392261814e8776c739a639be93b, 28e8761fd17d711f0365790f30eeb17a2a95a99e and 932fbd28177b391deff9d008b980f92ec53e97e0

Backports `stable/1.1.x`: f461c13ad5ca70b2b01a747a987765894a20255f, b9520f3db46e062aa3011f111ea452961f5d7f59 and 387315bebace1b1e836bef12aee831d40827ec85

**Pull request merged**